### PR TITLE
[PFS] Updates and typo fixes

### PR DESCRIPTION
**Pull request by mjmeans**

### CODE REVIEW

File: data/pathfinder_pfs/paizo/pathfinder_society/pfs_chronicle_sheets/pfscs_abilitycategories.lst [hunk 1]
```diff
@@ -11,208 +10,11 @@ ABILITYCATEGORY:PFSChronicle							VISIBLE:YES		EDITABLE:YES								CATEGORY:PFS
 ABILITYCATEGORY:Ally of Belhaim Perform Choice				VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:NO		FRACTIONALPOOL:NO		CATEGORY:Special Ability	TYPE:AllyOfBelhaimPerformChoice																										DISPLAYLOCATION:Chronicles
 
 # These ability pools hold the chronicle sheet boons
-ABILITYCATEGORY:Chronicle_S00-01						VISIBLE:QUALIFY	EDITABLE:YES								CATEGORY:Special Ability	TYPE:Chronicle_S00-01											DISPLAYNAME:PFS Scenario #01: Silent Tide											DISPLAYLOCATION:Chronicles
```
Comment (LegacyKing) [inline data/pathfinder_pfs/paizo/pathfinder_society/pfs_chronicle_sheets/pfscs_abilitycategories.lst]: When I see large chunks of data removed, it makes me nervous. What is the reasoning for this?

File: data/pathfinder_pfs/paizo/pathfinder_society/pfs_chronicle_sheets/online/thornkeep/pfscs_PZOGWK0001_equipmods.lst [hunk 2]
```diff
@@ -1,11 +1,9 @@
-# CVS $Revision: $ $Author: $ -- Tue Dec 15 01:48:08 2015 -- reformated by PCGen PrettyLST v6.06.00
-SOURCELONG:My Pathfinder Society Chronicle Sheets	SOURCESHORT:PFSCS	SOURCEWEB:http://www.paizo.com	SOURCEDATE:2014-08
+# CVS $Revision: $ $Author: mjmeans $ -- Tue Dec 15 01:48:08 2015 -- reformated by PCGen PrettyLST v6.06.00
```
Comment (LegacyKing) [inline data/pathfinder_pfs/paizo/pathfinder_society/pfs_chronicle_sheets/online/thornkeep/pfscs_PZOGWK0001_equipmods.lst]: Why did we do a file name change here?

File: data/pathfinder_pfs/paizo/pathfinder_society/pfs_chronicle_sheets/pfscs_equip.lst [hunk 3]
```diff
@@ -1,186 +1,13 @@
...
+Poison (Dusk Agate Extract)				OUTPUTNAME:[NAME]	TYPE:Goods.Poison.Consumable.PFS	COST:75	WT:0		SOURCEPAGE:p.558	SPROP:Contact; Fort DC 13; Freq 1 rnd (2); Effect unconsciousness for 1 min/1 Wis dmg; Cure 1 save	PREABILITY:1,CATEGORY=PFS Chronicle,Chronicle ~ S08-05
```
Comment (LegacyKing) [inline data/pathfinder_pfs/paizo/pathfinder_society/pfs_chronicle_sheets/pfscs_equip.lst]: Again, a lot of deletion going on without any reasoning.

**Comment by mjmeans** [comment]
I posted the reason for deletion of chronicle sheet file and items in the original file in exp. But I'll repost here for clarity and detail my reasoning for this. The file has been refactored so that each season's chronicle sheets and each multi-sheet module's chronicle sheets are separated into their own folder. This makes it easier for me to maintain since Paizo makes bulk changes on a season by season basis in addition to periodic changes in the Additional Resources guide. You will see also that there is a readme.md file for the pfs chronicles that lists any known issues with the chronicles which will help if another contributor wants to help and doesn't know where to start.

Additionally, I plan on implementing the seasonal changes of the Pathfinder Society Guild Guide (which is the newer versions of the Guide to Organized Play) in a similar way. There are two versions: 7.0 and 8.0 (representing seasons 7 and 8). The multi-sets Core Assumption and Complete will be renamed to include "(latest)" but the keys will remain the same. These will always use the most recent version. But if Paizo makes some critical change in a later version of the Guild Guide I want a way for players to open their character using an older version of Core Assumption and Complete. Each season changes in August of each year. This design will give the player the opportunity to gracefully resolve anything that may have broken from season to season.

**Pull request merged**